<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exportdlg</class>
 <widget class="QDialog" name="exportdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>353</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>347</width>
    <height>353</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>347</width>
    <height>353</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>321</width>
         <height>241</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="T">
        <attribute name="title">
         <string>Export image</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>12</y>
           <width>293</width>
           <height>231</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="maximumSize">
                 <size>
                  <width>54</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Format</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboFormat">
                <property name="maximumSize">
                 <size>
                  <width>91</width>
                  <height>30</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>png</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>bmp</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>jpg</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="maximumSize">
                 <size>
                  <width>65</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="text">
                 <string>File path</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="leFilePath">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="minimumSize">
                 <size>
                  <width>16</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="maximumSize">
                 <size>
                  <width>73</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>File name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="leFileName">
                <property name="maximumSize">
                 <size>
                  <width>142</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Quality</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="qualitySlider">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Dimensions</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboDimensions">
                <property name="minimumSize">
                 <size>
                  <width>131</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>131</width>
                  <height>30</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>800x600</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>1200x800</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2400x1600</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Data range</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboRange">
                <property name="minimumSize">
                 <size>
                  <width>131</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>131</width>
                  <height>30</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>Current view</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>All history</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Export data</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>301</width>
           <height>231</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="maximumSize">
                 <size>
                  <width>54</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Format</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboFormat_2">
                <property name="maximumSize">
                 <size>
                  <width>91</width>
                  <height>30</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>csv</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>txt</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="maximumSize">
                 <size>
                  <width>65</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="text">
                 <string>File path</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="leFilePath_2">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="minimumSize">
                 <size>
                  <width>16</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="maximumSize">
                 <size>
                  <width>73</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>File name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="leFileName_2">
                <property name="maximumSize">
                 <size>
                  <width>142</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Data range</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboRange_2">
                <property name="minimumSize">
                 <size>
                  <width>131</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>131</width>
                  <height>30</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>Current view</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>All history</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>188</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnExport">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
